lib/src/components/NxTransferList/__tests__/TransferListHalf.test.tsx

```diff
@@ -80,7 +80,7 @@ describe('TransferListHalf', function() {
     expect(onMoveAll).toHaveBeenCalledTimes(1);
   });
 
-  it('passes onMoveAll a set containing every id if no filterValue is set', function() {
+  it('passes onMoveAll a set containing every id if no filterValue is an array', function() {
```

Comment:
This change doesn't make sense.  "set" here was the verb - to _set_ a value.

Reply:
> This change doesn't make sense. "set" here was the verb - to _set_ a value.

ops! Fixed it here: 774e9702b6a1123bac0705354974edd6087775a5